src/test/java/com/rocketden/main/service/GameServiceTests.java

```diff
@@ -1,5 +1,6 @@
 package com.rocketden.main.service;
 
+import com.rocketden.main.config.WebSocketConfig;
 import com.rocketden.main.controller.v1.BaseRestController;
```

Comment:
You don't use `WebSocketConfig` or `BaseRestController`, so remove these imports.

Similar to before, do you get these warnings?

<img width="886" alt="Unused Import Warnings" src="https://user-images.githubusercontent.com/7987279/96641336-6c5a6a00-12d9-11eb-9f07-123e326ed9bd.png">